src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -124,8 +127,26 @@ class GcsPlacementGroup {
   /// Returns whether or not this is a detached placement group.
   bool IsDetached() const;
 
+  const rpc::PlacementGroupStats &GetStats() const;
+
+  rpc::PlacementGroupStats *GetMutableStats();
+
  private:
   FRIEND_TEST(GcsPlacementGroupManagerTest, TestPlacementGroupBundleCache);
+
+  /// Setup states other than placement_group_table_data_.
+  void SetupStates() {
+    auto stats = placement_group_table_data_.mutable_stats();
+    // If the value has never been set, it is 0.
+    if (stats->creation_request_received_ns() == 0) {
+      auto now = absl::GetCurrentTimeNanos();
+      stats->set_creation_request_received_ns(now);
+    }
+    if (stats->scheduling_state() == 0) {
```

Comment:
0 looks bad here?

Reply:
```
Bar foo() const: Returns the current value of the field. If the field is not set, returns the default value (0).
```

According to the doc, the default value is 0. I can alternatively just use `rpc::PlacementGroupStats::QUEUED`, but I thought it might be more confusing (like people will wonder why we set this and probably just remove it). I will probably just add a comment instead (that 0 is the default)